lib/auth/auth.go

```diff
@@ -6051,6 +6051,12 @@ func (a *Server) GetAccessCapabilities(ctx context.Context, req types.AccessCapa
 	return caps, nil
 }
 
+// GetRemoteAccessCapabilities requires information more readily available in
+// [ServerWithRoles], so the main implementation is there.
+func (a *Server) GetRemoteAccessCapabilities(context.Context, types.RemoteAccessCapabilitiesRequest) (*types.RemoteAccessCapabilities, error) {
+	return nil, trace.NotImplemented("GetRemoteAccessCapabilities not implemented")
+}
```

Comment:
is this easily avoidable or does it really need to be added to the AuthService interface